CRM/Financial/DAO/FinancialTrxn.php

```diff
@@ -340,6 +365,15 @@ static function &fields() {
             'type' => 'Text',
           ) ,
         ) ,
+        'financial_trxn_credit_card_number' => array(
```

Comment:
can we call this credit_card_pan instead? we don't want to imply the number is stored